.github/workflows/build_frontend-ng.yml

```
@@ -0,0 +1,39 @@
+name: Build and Deploy Frontend-NG
+
+on:
+  push:
+    branches:
+      - main # Or your main development branch
```

We have not renamed the default branch yet.

```suggestion
      - master
```

---

ack